doc/operation-and-maintenance/known-issues.md

```diff
@@ -10,3 +10,16 @@ MongooseIM will not connect to MySQL over TLS on OTP 20.3 due to [the MySQL driv
 * Upgrade OTP to 21.0 or higher.
 * Use unencrypted communication with MySQL.
 
+## MSSQL connectivity via ODBC
+
+ODBC driver currently used by MongooseIM is known to work only with Ubuntu Xenial x64 and FreeTDS 0.91-6.1build1.
```

Comment:
Perhaps this is a case of an article drop -> `The ODBC driver...`.

Reply:
The ODBC driver currently used by MongooseIM is known to only work with Ubuntu Xenial x64 and FreeTDS 0.91-6.1build1.